packages/webapp/src/components/Card/NoSearchResults/index.jsx

```diff
@@ -0,0 +1,56 @@
+/*
+ *  Copyright 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './styles.module.scss';
+import { Semibold, Text } from '../../Typography';
+import { useTranslation, Trans } from 'react-i18next';
+import clsx from 'clsx';
+
+export const NoSearchResults = ({
+  searchTerm = '',
+  includeFiltersInClearSuggestion = false,
```

Comment:
Could add default props to the component too

```js
NoSearchResults.defaultProps = {
    includeFiltersInClearSuggestion: false
}
```

Reply:
Thank you! I'll merge to get the code available for our two WIP dashboard tickets, but I've made a note to fix both PropTypes on the next search PR.